omod/src/main/java/org/openmrs/module/fhir2/web/servlet/FhirRestServlet.java

```diff
@@ -58,6 +59,10 @@
 	@Qualifier("hapiLoggingInterceptor")
 	private LoggingInterceptor loggingInterceptor;
 	
+	@Autowired
+	@Qualifier("hapiSummaryInterceptor")
```

Comment:
Can't we just call this "summaryInterceptor"?

Reply:
We also don't necessarily need to use Spring for this. `new SummaryInterceptor()` is fine.

Reply:
@ibacher I'll make the mentioned changes.

Reply:
@ibacher Will make the mentioned changes.